<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontPropertyWidget</class>
 <widget class="QWidget" name="FontPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="mainLayout" stretch="1,0,0,0,0,0,0,0">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QComboBox" name="FontFamily">
       <property name="toolTip">
        <string>Select font</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <property name="minimumContentsLength">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="FontSize">
       <property name="toolTip">
        <string>Set font size</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="pqColorChooserButtonWithPalettes" name="FontColor">
       <property name="toolTip">
        <string>Set font color</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="Opacity">
       <property name="toolTip">
        <string>Set font opacity</string>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="Bold">
       <property name="toolTip">
        <string>Bold</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Components/Resources/pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqBold24.png</normaloff>:/pqWidgets/Icons/pqBold24.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="Italics">
       <property name="toolTip">
        <string>Italics</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Components/Resources/pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqItalics24.png</normaloff>:/pqWidgets/Icons/pqItalics24.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="Shadow">
       <property name="toolTip">
        <string>Shadow</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Components/Resources/pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqShadow24.png</normaloff>:/pqWidgets/Icons/pqShadow24.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="Justification">
       <property name="text">
        <string>...</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="pqFileChooserWidget" name="FontFile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Specify the path to a TTF file here.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqColorChooserButtonWithPalettes</class>
   <extends>QToolButton</extends>
   <header>pqColorChooserButtonWithPalettes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqFileChooserWidget</class>
   <extends>QLineEdit</extends>
   <header>pqFileChooserWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
